azure-pipelines.yaml

```diff
@@ -112,3 +121,17 @@ stages:
               pr_number: $(pull_request_number)
               merge_method: 'merge'
             displayName: Merge PR# $(System.PullRequest.PullRequestNumber)
+
+  - stage: PoliCheck
+    dependsOn: ['Windows', 'Mac']
```

Comment:
This is not needed and will just extend the time to completion

Reply:
Done.